test/datalog_tests/api_test/README

```diff
@@ -0,0 +1 @@
+A client program to [`copy.dl`](../copy.dl). Compile the DDlog code (e.g. with stack test --ta "-p copy") before compiling this client. 
```

Comment:
I would call this file README.md; then it displays nicely on github.
Perhaps it's better to point to the run.sh script in the readme too.